cmd/mcu_com.c

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-2.0+
+
+#include <common.h>
+#include <command.h>
+#include <console.h>
+
+__weak int board_mcu_set_cpu_ready(void)
+{
+	return 0;
```

Comment:
Functions return _OK_ even if they do not do anything? Should we fail here and indicate this is to be overridden by derivative boards?

Reply:
I agree, taken from 9f0e582239916661ca3faa64ca9a7788f80990d9. What do you think @htho-lgs?

Reply:
Yes - we should return error.